### PR TITLE
No2 Update NoBenefit text

### PR DESCRIPTION
**Pull request by arranfw**
<html>
<body>
<google-sheets-html-origin>

Change # | Summary | Description
-- | -- | --
2 | "Not likely to benefit from antivirals" text has been changes | Update the text to match the new text when a user is not likely to benefit from antivirals for questions 1 and 3. Other similar ends to the journey have a different text

</body>
</html>

image
image

### CODE REVIEW

File: components/EndOfJourney.tsx [hunk 1]
```diff
@@ -1,13 +1,99 @@
...
+
+const NoBenefit: React.FC = () => (
+  <EndOfJourneyContainer>
+    <EndOfJourneyTitle>
+      Based on your answer you would likely not benefit from antivirals for COVID-19 at this time.
+    </EndOfJourneyTitle>
+    <p>
+      These medications have been approved specifically for people experiencing mild-moderate
+      COVID-19 symptoms and require a positive test to be confirmed. As with most medications, there
+      can be side effects when they are not used for their intended purpose.If you are experiencing
```
Comment (fw-noel) [inline components/EndOfJourney.tsx:43]: Space after period

File: components/EndOfJourney.tsx [hunk 2]
```diff
@@ -1,13 +1,99 @@
...
+  </EndOfJourneyContainer>
+);
+
+const TooManyDays: React.FC = () => (
+  <EndOfJourneyContainer>
+    <p>
+      In order to be effective, the antiviral treatments that are currently available need to be
+      given within the first 5-7 days of symptom onset. Stay home and away from others until you
+      feel well enough to return to your regular activities. You should continue to seek medical
+      care if you feel you need it. For more information, go to&nbsp;
```
Comment (fw-noel) [inline components/EndOfJourney.tsx:73]: &#8203; earlier, &nbsp; here. I may not fully know what 8203 does

File: components/EndOfJourney.tsx [hunk 3]
```diff
@@ -1,13 +1,99 @@
...
+  </EndOfJourneyContainer>
+);
+
+const AntiviralBenefit: React.FC = () => (
+  <EndOfJourneyContainer>
+    <EndOfJourneyTitle>You may benefit from antiviral treatment.</EndOfJourneyTitle>
+    <p>
+      Please call{' '}
+      <a className='text-bcBlueLink font-bold' href='tel:811'>
+        8-1-1
```
Comment (fw-noel) [inline components/EndOfJourney.tsx:92]: Should be ServiceBC link

**Review by fw-noel — CHANGES_REQUESTED** [review]

File: components/EndOfJourney.tsx (hunk 3 above)
Comment (arranfw) [reply components/EndOfJourney.tsx:92]: Updated in #10

**Review by fw-noel — APPROVED** [review]
I see you did this in No6

File: components/EndOfJourney.tsx (hunk 2 above)
Comment (arranfw) [reply components/EndOfJourney.tsx:73]: Ahh yeah this is a zero width space so we don't get a small space when the link causes a break 
image
image

**Pull request merged**